CPT331.WebAPI/CPT331.WebAPI.csproj

```diff
@@ -125,8 +143,27 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Content\bootstrap-theme.css" />
```

Comment:
I'm not sure if swagger needs these. I added the missing libraries to fix the Home view.